<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RimFacePanel</class>
 <widget class="QWidget" name="RimFacePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>249</width>
     <height>394</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Rim Face Piece</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="edgeBtn">
           <property name="text">
            <string>Edge Based</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">piecePositionGrp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="faceBtn">
           <property name="text">
            <string>Face Based</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">piecePositionGrp</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="pieceShapeGrpBox">
         <property name="title">
          <string>Piece Shape</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QRadioButton" name="rectBtn">
            <property name="text">
             <string>Rectangle</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">shapeGrp</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="diamBtn">
            <property name="text">
             <string>Diamond</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">shapeGrp</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="smoothBox">
         <property name="text">
          <string>Smooth on Edge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QRadioButton" name="halfBtn">
           <property name="text">
            <string>Half</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pieceSizeGrp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="wholeBtn">
           <property name="text">
            <string>Whole</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pieceSizeGrp</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="connectorBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Add Connector</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Beizier</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QRadioButton" name="quaBtn">
           <property name="text">
            <string>Quadratic</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bezierGrp</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cubicBtn">
           <property name="text">
            <string>Cubic</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">bezierGrp</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Roundness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="widthSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="widthText">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Thickness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="heightSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="heightText">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="intersectionBox">
      <property name="text">
       <string>Avoid Self Intersection</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="setBridgerBtn">
          <property name="text">
           <string>Set Bridger...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="okBtn">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="piecePositionGrp"/>
  <buttongroup name="bezierGrp"/>
  <buttongroup name="pieceSizeGrp"/>
  <buttongroup name="shapeGrp"/>
 </buttongroups>
</ui>
